Documentatie uit issue 508 toegevoegd en rule gedefinieerd. (#512)
Geonovum/KP-APIs: API-strategie-modulen/API-strategie-mod-geo/crs.md

- The default CRS, CRS84, is listed first in the list of supported CRSs in the API; if the consumer does not specify the CRS it is assumed it uses the default.
- Coordinate reference systems API strategy: request/response in RD; ETRS89; CRS84; Pseudo  Mercator;
- Consider no-regret: record in multiple much-requested CRSs instead of on-the-fly transformation;
- Use RDNAPTRANS™ 2018 to transform Amersfoort / RD New to ETRS89 (correction grid);
- Presentation depending on context (e.g. user requirements);
- Exchange format (notation) ETRS89 and WGS84 longitude latitude in decimal degrees: DD.ddddddddd (for example: `5.962376256, 52.255023450`)
- Exchange format (notation) RD and Pseudo Mercator X Y in meters: `195427.5200 311611.8400`
- Use the CRS Guidelines [[hr-crs]] for coordinate transformations.
- CRSs may be grouped into ensemble CRSs. When exchanging geometry an ensemble member CRS shall be used (instead of an ensemble CRS) where possible.

> - Use enemble member CRS (instead of an ensemble CRS) as output of coordinate transformation, where possible.

Reviewer: ```suggestion
- Use an ensemble member CRS (instead of an ensemble CRS) as output of coordinate transformation, where possible.
```

> - APIs shall support and advertise both ensemble CRSs and ensemble member CRSs if geometry is exchanged and the CRS for the geometry is a member CRS.

Reviewer: ```suggestion
- APIs shall support and advertise both ensemble CRSs and ensemble member CRSs if geometry is exchanged and the CRS for the geometry is an ensemble member CRS.
```

> - If multiple coordinate transformation steps are required than APIs shall support and advertised all CRSs that are output of each CRS transformation.

Reviewer: ```suggestion
- If multiple coordinate transformation steps are required than APIs shall support and advertise all CRSs that are output of each CRS transformation step.
```

Author: Ik heb deze zin herschreven zodat deze overeenkomt met wat er in #508 en par. [3.3.2.2 Handreiking Gebruik coördinaatreferentiesystemen bij uitwisseling en visualisatie van geo-informatie](https://docs.geostandaarden.nl/crs/crs/#transformatie-tussen-etrs89-en-itrs-wgs-84) staat.
